Update the definition of the `authenticatedUser` entry (#3099)
neo4j/docs-operations: modules/ROOT/pages/monitoring/logging.adoc

This field is optional and thus will not be populated for all security events.

| executingUser

> | The username of the user triggering the security event.

Author: This is a security log entry. I assume we log usernames not names.

| The database name on which the query is run. This field will be `<none>` if the query cannot be parsed and routed to a database.

| executingUser
| The username of the user executing the query.
Either same as `authenticatedUser` or an impersonated user.

> | authenticatedUser footnote:[If SSO is enabled, the username is defined by the xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.claims.username[`dbms.security.oidc.<provider>.claims.username`] setting.

Reviewer: This is more complicated than that, because if you have local users `dbms.security.require_local_user=true` then you will get the username assigned to the SSO user.

> | authenticatedUser footnote:[If SSO is enabled, the username is defined by the xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.claims.username[`dbms.security.oidc.<provider>.claims.username`] setting.

Author: ```suggestion
| authenticatedUser footnote:[When SSO is enabled, the username is determined either by the xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.claims.username[`dbms.security.oidc.<provider>.claims.username`] setting or, if local user mapping is required (`dbms.security.require_local_user=true`), by the corresponding mapped local username .
```

> If LDAP is configured, Neo4j determines the username string using one of two settings: xref:configuration/configuration-settings.adoc#config_dbms.security.ldap.authentication.user_dn_template[`dbms.security.ldap.authentication.user_dn_template`] or xref:configuration/configuration-settings.adoc#config_dbms.security.ldap.authentication.attribute[`dbms.security.ldap.authentication.attribute`].

Reviewer: LDAP can use linked users as well.

| The database name on which the query is run. This field will be `<none>` if the query cannot be parsed and routed to a database.

| executingUser
| The username of the user executing the query.
Either same as `authenticatedUser` or an impersonated user.

> | authenticatedUser footnote:[When SSO is enabled, the username is determined either by the xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.claims.username[`dbms.security.oidc.<provider>.claims.username`] setting.

Author: ```suggestion
| authenticatedUser footnote:[When SSO is enabled, the username is determined by the xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.claims.username[`dbms.security.oidc.<provider>.claims.username`] setting.
```

If LDAP is configured, Neo4j determines the username string using one of two settings: xref:configuration/configuration-settings.adoc#config_dbms.security.ldap.authentication.user_dn_template[`dbms.security.ldap.authentication.user_dn_template`] or xref:configuration/configuration-settings.adoc#config_dbms.security.ldap.authentication.attribute[`dbms.security.ldap.authentication.attribute`].

> For both SSO and LDAP, if local user mapping is required (`dbms.security.require_local_user=true`), the log records the mapped local username instead of the external identity string.

Author: @mnd999, I updated the PR. Could you take a look?

Reviewer: I might say OIDC here instead of SSO but yeah it's accurate.

> For both SSO and LDAP, if local user mapping is required (`dbms.security.require_local_user=true`), the log records the mapped local username instead of the external identity string.

Author: ```suggestion
For both OIDC and LDAP, if local user mapping is required (`dbms.security.require_local_user=true`), the log records the mapped local username instead of the external identity string.
```
